app/helpers/dashboard_helper.rb

```diff
@@ -68,4 +68,12 @@ def six_month_rate_change(facility, rate_name)
   def facility_size_six_month_rate_change(facility_size_data, rate_name)
     facility_size_data[@period][rate_name] - facility_size_data[@start_period][rate_name] || 0
   end
+
+  def registrations_end_date
+    if @period.value.month == Date.current.month
```

Comment:
`.month` just gives you a number between 1 and 12, so this method will behave unexpectedly if we look at, for example, last February's report today.

Can we, instead, do something like this:

```ruby
if @period.end_date > Date.current
  Date.current.to_s(:day_mon_year)
else
  @period.bp_control_range_end_date
end
```